AIDevGallery/Pages/Scenarios/ScenarioPage.xaml.cs

```diff
@@ -137,6 +160,15 @@ private async void ModelSelectionControl_SelectedModelChanged(object sender, Mod
         {
             foreach (var s in samples)
             {
+                if (selectedModelDetails.HardwareAccelerators.Contains(HardwareAccelerator.OLLAMA))
```

Comment:
What is this comparing? Only the hardware accelerator? This seems wrong, but I'm not sure.

Reply:
The modelDetails for Ollama models are not in the dictionary of models so the id will never be found with GetModelDetails if the selected model is Ollama, so this just checks if the selected model is Ollama and if the sample calls for LanguageModel